sklearn/metrics/classification.py

```diff
@@ -2038,3 +2038,110 @@ def brier_score_loss(y_true, y_prob, sample_weight=None, pos_label=None):
     y_true = np.array(y_true == pos_label, int)
     y_true = _check_binary_probabilistic_predictions(y_true, y_prob)
     return np.average((y_true - y_prob) ** 2, weights=sample_weight)
+
+
+def fall_out(y_true, y_pred, labels=None, sample_weight=None):
+    """compute the false positive rate
+       False positive rate indicate the
+       number of sample which are classified as
+       negative but they are positive .In
+       other word we can say that it is the
+       chances of occuring a TYPE-1 error
+
```

Comment:
typo: 'positive .In' -> 'positive. In'
Also please add a period at the end of your sentence.